app/src/main/java/com/compscieddy/meetinthemiddle/activity/OnboardingActivity.java

```diff
@@ -0,0 +1,78 @@
+package com.compscieddy.meetinthemiddle.activity;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.view.View;
+
+import com.compscieddy.meetinthemiddle.R;
+import com.compscieddy.meetinthemiddle.fragment.OnboardingFragment1;
+import com.compscieddy.meetinthemiddle.fragment.OnboardingFragment2;
+import com.compscieddy.meetinthemiddle.fragment.OnboardingFragment3;
+
+/**
+ * Created by Ant Henderson on 6/15/16.
+ * anthenderson89@gmail.com
+ */
+public class OnboardingActivity extends FragmentActivity {
+    private ViewPager pager;
+    private FloatingActionButton onboardingFab;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_onboarding);
+
+        pager = (ViewPager)findViewById(R.id.onboardingPager);
+        onboardingFab = (FloatingActionButton)findViewById(R.id.onboardingFab);
+
+        FragmentStatePagerAdapter adapter = new FragmentStatePagerAdapter(getSupportFragmentManager()) {
+            @Override
+            public Fragment getItem(int position) {
+                switch (position) {
+                    case 0 : return new OnboardingFragment1();
```

Comment:
This can be refactored to have 1 OnboardingFragment and then just pass in the index (1,2,3) as a parameter